model/gen-ai/registry.yaml

```diff
@@ -332,3 +332,57 @@ groups:
 
           Additional output format details may be recorded in the future in the
           `gen_ai.output.{type}.*` attributes.
+      - id: gen_ai.evaluation.name
+        stability: development
+        type: string
+        brief: The qualified name of the evaluator used to evaluate the GenAI response.
+        examples: ["Relevance", "IntentResolution"]
+        note: >
+          This attribute identifies the evaluation name.
+      - id: gen_ai.evaluation.score
+        stability: development
+        type: double
+        brief: The evaluation score for the GenAI response.
+        examples: [4.0]
+      - id: gen_ai.input.metadata
```

Comment:
do we have a real-life example on where/how it would be useful?

Reply:
I have moved this as a span attribute and added an example. Please share your feedback.